core/lib/multivm/src/versions/vm_latest/bootloader/utils.rs

```diff
@@ -11,16 +11,27 @@ use crate::{
         bootloader::l2_block::BootloaderL2Block,
         constants::{
             get_bootloader_tx_description_offset, get_compressed_bytecodes_offset,
-            get_operator_provided_l1_messenger_pubdata_offset, get_operator_refunds_offset,
-            get_tx_description_offset, get_tx_operator_l2_block_info_offset,
-            get_tx_overhead_offset, get_tx_trusted_gas_limit_offset,
-            BOOTLOADER_TX_DESCRIPTION_SIZE, OPERATOR_PROVIDED_L1_MESSENGER_PUBDATA_SLOTS,
+            get_current_number_of_roots_in_block_offset, get_interop_blocks_begin_offset,
+            get_interop_root_offset, get_operator_provided_l1_messenger_pubdata_offset,
+            get_operator_refunds_offset, get_tx_description_offset,
+            get_tx_operator_l2_block_info_offset, get_tx_overhead_offset,
+            get_tx_trusted_gas_limit_offset, BOOTLOADER_TX_DESCRIPTION_SIZE,
+            INTEROP_ROOT_SLOTS_SIZE, OPERATOR_PROVIDED_L1_MESSENGER_PUBDATA_SLOTS,
             TX_OPERATOR_SLOTS_PER_L2_BLOCK_INFO,
         },
         MultiVmSubversion,
     },
 };
 
+pub(super) struct L2BlockApplicationConfig {
+    pub txs_index: usize,
+    pub start_new_l2_block: bool,
+    pub subversion: MultiVmSubversion,
+    pub apply_interop_roots: bool,
+    pub preexisting_interop_roots_number: usize,
+    pub preexisting_blocks_number: usize,
```

Comment:
Config looks overly complex, why do we need `apply_interop_roots`, can't just empty interop root array can be provided?